pkg/appconsts/appconsts.go

```diff
@@ -87,6 +87,10 @@ var (
 	// EvidenceNamespaceID is the namespace reserved for evidence.
 	EvidenceNamespaceID = namespace.ID{0, 0, 0, 0, 0, 0, 0, 3}
 
+	// TailTransactionPaddingNamespaceId is the namespace used for padding after
+	// all transactions (ordinary and PFBs) but before blobs.
+	TailTransactionPaddingNamespaceID = namespace.ID{0, 0, 0, 0, 0, 0, 0, 255}
```

Comment:
Q: We don't want it to just increment from the previous one i.e. 4?

Reply:
I think it would be preferable if TailTransactionPaddingNamespaceID was the maximum reserved namespace (i.e. `0, 0, 0, 0, 0, 0, 0, 255`) because it leaves us the option of introducing new reserved namespaces in the 4-255 range in the future.